app/api/events.py

```diff
@@ -210,15 +278,33 @@ def before_get_object(self, view_kwargs):
                 else:
                     view_kwargs['id'] = None
 
-    decorators = (is_coorganizer, )
+    def after_get(self, result):
+        result = changeToTimezone(result)
```

Comment:
@shubham-padia @niranjan94 
I think now I understand what's happening.
See this ---
```
postgres=# show timezone;
 TimeZone 
----------
 UTC
(1 row)

postgres=# 
```

So the server timezone ( in this case my PC ) has **UTC timezone** in **postgresql** .
So upto here it's all correct.
Only thing I can't check is what is actually being saved and that is causing the error